locale/lt/LC_MESSAGES/crm.po

```diff
@@ -247,9 +249,9 @@ msgid ""
 "unplanned opportunities and are displayed in grey in kanban progress bar."
```

Comment:
it should be gray, not grey. Cause default lang is US en (grey is used in British English).